ovs-p4rt/ovs_p4rt.cc

```diff
@@ -1432,6 +2027,17 @@ absl::Status ConfigTunnelTermTableEntry(ovs_p4rt::OvsP4rtSession* session,
 // Functions with C interfaces
 //----------------------------------------------------------------------
 
+enum ovs_tunnel_type TunnelTypeStrtoEnum(const char* tnl_type) {
```

Comment:
looks like no caller to this API, is this needed?

Reply:
This function is called from ovs code while filling tunnel type.